Tests/DeepLink/DPLDeepLink_AppLinksSpec.m

```diff
@@ -1,5 +1,8 @@
-@import DeepLinkKit.Private;
-@import DeepLinkKit.AppLinks;
+#import "DPLDeepLink.h"
```

Comment:
Well, you haven't got subspecs defined, so I don't see the point... CP generates an umbrella header for it all and you can do `@import DeepLinkKit.AppLinks;`. I updated the test to use it like that.